sklego/meta/ordinal_classification.py

```diff
@@ -178,8 +205,29 @@ def _fit_binary_estimator(self, X, y, y_label):
             The fitted binary classifier.
         """
         y_bin = (y <= y_label).astype(int)
-        fitted_model = clone(self.estimator).fit(X, y_bin)
         if self.use_calibration:
-            return CalibratedClassifierCV(fitted_model, cv="prefit").fit(X, y_bin)
+            return CalibratedClassifierCV(estimator=clone(self.estimator), **self.calibrarion_kwargs).fit(X, y_bin)
         else:
-            return fitted_model
+            return clone(self.estimator).fit(X, y_bin)
+
+    def score(self, X, y):
+        """Returns the accuracy score on the given test data and labels.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features )
+            The training data.
+        y : array-like of shape (n_samples,)
+            The target values.
+
+        Returns
+        -------
+        score : float
+            Accuracy score of self.predict(X) wrt. y.
+        """
+        return accuracy_score(y, self.predict(X))
```

Comment:
Doesn't the `ClassifierMixin` provide this already? If we want to do something custom here, perhaps it's better to re-use the `score()` method of the underlying classifier?

Reply:
Oh you are right! I am removing it for now 😊
Not sure what something custom and default would be.

I am personally not paying much attention to accuracy in the ordinal problem I am dealing with, but don't want to put anything very opinionated, so let's go with the hierarchical implementation from `ClassifierMixin`